<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JoinProjectWindow</class>
 <widget class="QDialog" name="JoinProjectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>View Project</string>
  </property>
  <widget class="QLabel" name="projectNameLbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>     Project Name Goes Here</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>641</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTextBrowser" name="projectDescriptTb">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>571</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLbl">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Description:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="leaveProjectBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="joinProjectBtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>310</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Join</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLbl">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
